src/Orchard.Web/Core/Contents/Controllers/AdminController.cs

```diff
@@ -477,6 +482,60 @@ public class AdminController : ContentControllerBase, IUpdateModel {
         void IUpdateModel.AddModelError(string key, LocalizedString errorMessage) {
             ModelState.AddModelError(key, errorMessage.ToString());
         }
+
+        /// <summary>
+        /// Set VisibleAtFrontEnd flag for menuPart
+        /// </summary>
+        /// <param name="menu">The menu item</param>
+        /// <param name="value">Show->true or Hide->false</param>
+        private void VisibleAtFrontEnd(MenuPart menu, Boolean value) {
```

Comment:
Should this method be here? It's working on MenuPart, but this controller is the one for ContentItems